oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java

```diff
@@ -193,17 +193,37 @@ public static GoogleCredentials getApplicationDefault(HttpTransportFactory trans
   }
 
   /**
-   * Returns credentials defined by a JSON file stream.
+   * This method is obsolete because of a potential security risk. Use the credential specific load
+   * method instead
+   *
+   * <p>Returns credentials defined by a JSON file stream.
```

Comment:
Can we combine the two added sections together?

Reply:
Done. I added the "Returns credentials" sentence before the obsolete method sentence, in a single paragraph.

Reply:
Apologies, I meant can you move these lines to below the 
```
<p>The stream can contain a Service Account key file in JSON format from the Google Developers Console or a stored user credential using the format supported by the Cloud SDK.
 ```
 line? Or move L204-L226 up?
 
 So that the the warnings about the security vulnerability is together?